### PR TITLE
Copy-pasta static HTML for Personal Information.

### PR DESCRIPTION
**Pull request by awong-dev**
This provides a baseline to work off of.

### CODE REVIEW

File: _health-care/_js/_components/personal-information-panel.jsx [hunk 1]
```diff
@@ -1,44 +1,26 @@
...
-import FullName from './full-name';
-import MothersMaidenName from './mothers-maiden-name';
+import AdditionalInformationSection from './additional-information-section';
+import DemographicInformationSection from './demographic-information-section';
+import NameAndGeneralInfoSection from './name-and-general-info-section';
+import VaInformationSection from './va-information-section';
+import VeteranAddressSection from './veteran-address-section';
 
 class PersonalInformationPanel extends React.Component {
-  constructor() {
```
Comment (akainic) [inline _health-care/_js/_components/personal-information-panel.jsx]: Wait, we can delete this?

File: _health-care/_js/_components/name-and-general-info-section.jsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            <label htmlFor="veteran_gender">Gender</label>
+            <select name="veteran[gender]" ><option value="M">Male</option>
+              <option value="F">Female</option>
+              <option value="2">NeedOtherOfficialOptions</option>
+            </select>
+
+            <h4>Place of Birth</h4>
+            <div>
+              <div className="usa-input-grid usa-input-grid-medium">
+                <label htmlFor="veteran_city_of_birth">City</label>
```
Comment (akainic) [inline _health-care/_js/_components/name-and-general-info-section.jsx]: We'll need to make sure all the labels and inputs are associated eventually, I can create a Waffle ticket to check those

**Comment by akainic** [comment]
Should we eventually create subdirectories in the `_components/` directory to organize some of the components by topic?

File: _health-care/_js/_components/name-and-general-info-section.jsx (hunk 2 above)
Comment (awong-dev) [inline _health-care/_js/_components/name-and-general-info-section.jsx]: Yes. I know these are all broken. I actually think as we go to create a <ValidatedInput> component that will handle all this for us.

**Comment by awong-dev** [comment]
Yes...but maybe after I finish this?

**Comment by awong-dev** [comment]
TBR=@akainic

Pre-emptively merging since this CL is pretty much completely copy-pasta so any sort of review is just a rubber-stamp.

**Pull request merged**